### PR TITLE
Fix double mallet start point collision

### PR DESCRIPTION
**Pull request by berkgungor**
- updated initial pose for demo.launch to avoid collision

### CODE REVIEW

File: marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf [hunk 1]
```diff
@@ -96,15 +96,6 @@
         <joint name="ur5_wrist_2_joint" value="1.6173"/>
         <joint name="ur5_wrist_3_joint" value="0"/>
     </group_state>
-    <group_state name="marimbabot_home" group="arm_mallets">
```
Comment (Flova) [inline marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf:99]: Why did you delete the home position? It is utilized at multiple points in the planning pipeline?

Comment (berkgungor) [reply marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf:99]: two marimbabot_home was defind and looked like a duplicate. but just saw that one line is different "mallet_finger" in second marimbabot_home group. I think we should delete the first defined group.

Comment (Flova) [reply marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf:99]: We can keep both as they are configurations for different groups. One is for "arm_mallets" and includes a state for the `mallet_finger` and one is only for the `arm` (the old single mallet setup). We can keep both as they do not result in any problems and define a homing position for both groups.

Comment (Flova) [reply marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf:99]: I would revert [8997cc3](https://github.com/UHHRobotics22-23/marimbabot/pull/268/commits/8997cc3b00e4c0c0dedd05830bb06b81417c6b18)

File: marimbabot_ur5_moveit_config/config/fake_controllers.yaml [hunk 2]
```diff
@@ -26,4 +26,4 @@ initial:  # Define initial robot poses per group
   - group: gripper
     pose: closed
   - group: arm
-    pose: home
\ No newline at end of file
+    pose: marimbabot_home
```
Comment (Juphex) [inline marimbabot_ur5_moveit_config/config/fake_controllers.yaml:29]: looks good

**Review by Juphex — COMMENTED** [review]
works but please see https://github.com/UHHRobotics22-23/marimbabot/pull/268/files#r1341241545

**Review by Flova — APPROVED** [review]
